endpoints/openrtb2/video_auction.go

```diff
@@ -5,6 +5,7 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"github.com/prebid/prebid-server/privacy"
```

Comment:
FYI: Other editors group non-built in imports after the built-in ones. This is likely to be automatically overwritten in the future by other contributors. Does your IDE have that as an option?

Reply:
I think my IDE sorted it in alphabetical order. I'll fix this.